cookiecutter.json

```diff
@@ -0,0 +1,12 @@
+{
+  "author": "Spyder Project Contributors",
+  "email": "admin@spyder-ide.org",
+  "github_username": "spyder-ide",
+  "plugin_name": "Demo Plugin",
+  "repo_name": "spyder-{{ cookiecutter.plugin_name.lower().replace(' ', '-')}}",
+  "project_name": "{{ cookiecutter.repo_name.replace('-', '_') }}",
+  "description": "Plugin for Spyder IDE.",
+  "version": "0.1.0",
+  "create_config_page": "n",
+  "open_source_license": ["MIT license", "BSD license", "ISC license", "Apache Software License 2.0", "GNU General Public License v3", "Not open source"]
```

Comment:
What I am saying is why should we encourage use of any other license if MIT is perfectly fine.